app/controllers/sessions_controller.rb

```diff
@@ -1,22 +1,30 @@
 class SessionsController < ApplicationController
+
+  def new
+
+  end
+
   def create
-    session[:joke_id] = params[:joke_id]
-    if params[:joke_id]
-      redirect_to items_path
+    user ||=  User.find_by_email(params[:user][:email])
+    if user && user.authenticate(params[:user][:password])
+      session[:user] = user.id
+      flash[:notice] = "The Dad welcome you, #{user.user_name}."
+      redirect_to root_path
     else
-      user ||=  User.find_by_email(params[:user][:email])
-      if user && user.authenticate(params[:user][:password])
-        session[:user] = user.id
-        flash[:notice] = "Welcome: #{user.user_name}"
-        redirect_to controller: 'welcome', action: 'index'
-      else
-        flash[:notice] = "Invalid Login"
-        redirect_to action: 'new'
-      end
+      flash[:notice] = "Invalid Login"
+      redirect_to action: 'new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.